nvbios/bios.h

```diff
@@ -1611,8 +1611,37 @@ struct envy_bios_T {
 	struct envy_bios_T_unk0 unk0;
 };
 
+enum envy_bios_d_dp_info_link_rate {
+	ENVY_BIOS_DP_INFO_LINK_RATE_162 = 0x06,
+	ENVY_BIOS_DP_INFO_LINK_RATE_270 = 0x0a,
+	ENVY_BIOS_DP_INFO_LINK_RATE_540 = 0x14,
+	ENVY_BIOS_DP_INFO_LINK_RATE_810 = 0x1e,
+};
```

Comment:
FYI, this value seems to be the multiplicator that needs to be applied on the crystal clock (27MHz).

Reply:
Yes, for example 8.1G is `0x1e * 27,000 * 10 ("nr") = 8,100,000,000`

Reply:
It's not related to the crystal on the GPU at all.  The values come from the DP spec.